openedx/core/djangoapps/programs/migrations/0001_initial.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
Could you group the `__future__` imports into a single line in these migration files?  I think `isort` should take care of that.